src/send-request.js

```diff
@@ -39,12 +48,11 @@ export const xhr = ({ url, data, headers, options, captureMetrics, callback, ret
         ...options._metrics,
     })
 
+    headers = { ...headers, 'Content-Type': 'application/x-www-form-urlencoded' }
```

Comment:
Based on the code it seems this would blow up for blob requests.

Reply:
ah sorry left out the if statement